packages/react-tinacms-editor/src/translator/MarkdownTranslator/hugo-utils.ts

```diff
@@ -0,0 +1,37 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import { Token } from './types'
+import { Schema } from 'prosemirror-model'
+
+const RELREF_LINK_REGEX = /\[(.+)\]\((\{\{\<\srelref ".*"\s>\}\}\))/g
```

Comment:
While we're at it, can we also add support for `ref` here?
https://gohugo.io/content-management/cross-references/#use-ref-and-relref

Reply:
FYI Hugo has also pipes and filters, could that regexp also support those as in https://gohugo.io/functions/rellangurl/

```
{{ "blog/" | absLangURL }}
{{ "blog/" | relLangURL }}
```

Reply:
@DirtyF : I can add more regex rules for this. It will be useful to have some samples of valid url for Hugo to test the REGEX. It is required to ensure that it works correctly in all possible combinations.